libethdrivers/src/plat/imx6/uboot/phy.c

```diff
@@ -51,69 +45,81 @@
  */
 static int genphy_config_advert(struct phy_device *phydev)
 {
-	u32 advertise;
-	int oldadv, adv;
-	int err, changed = 0;
-
-	/* Only allow advertising what
-	 * this PHY supports */
-	phydev->advertising &= phydev->supported;
-	advertise = phydev->advertising;
-
-	/* Setup standard advertisement */
-	oldadv = adv = phy_read(phydev, MDIO_DEVAD_NONE, MII_ADVERTISE);
-
-	if (adv < 0)
-		return adv;
-
-	adv &= ~(ADVERTISE_ALL | ADVERTISE_100BASE4 | ADVERTISE_PAUSE_CAP |
-		 ADVERTISE_PAUSE_ASYM);
-	if (advertise & ADVERTISED_10baseT_Half)
-		adv |= ADVERTISE_10HALF;
-	if (advertise & ADVERTISED_10baseT_Full)
-		adv |= ADVERTISE_10FULL;
-	if (advertise & ADVERTISED_100baseT_Half)
-		adv |= ADVERTISE_100HALF;
-	if (advertise & ADVERTISED_100baseT_Full)
-		adv |= ADVERTISE_100FULL;
-	if (advertise & ADVERTISED_Pause)
-		adv |= ADVERTISE_PAUSE_CAP;
-	if (advertise & ADVERTISED_Asym_Pause)
-		adv |= ADVERTISE_PAUSE_ASYM;
-
-	if (adv != oldadv) {
-		err = phy_write(phydev, MDIO_DEVAD_NONE, MII_ADVERTISE, adv);
-
-		if (err < 0)
-			return err;
-		changed = 1;
-	}
-
-	/* Configure gigabit if it's supported */
-	if (phydev->supported & (SUPPORTED_1000baseT_Half |
-				SUPPORTED_1000baseT_Full)) {
-		oldadv = adv = phy_read(phydev, MDIO_DEVAD_NONE, MII_CTRL1000);
-
-		if (adv < 0)
-			return adv;
-
-		adv &= ~(ADVERTISE_1000FULL | ADVERTISE_1000HALF);
-		if (advertise & SUPPORTED_1000baseT_Half)
-			adv |= ADVERTISE_1000HALF;
-		if (advertise & SUPPORTED_1000baseT_Full)
-			adv |= ADVERTISE_1000FULL;
-
-		if (adv != oldadv) {
-			err = phy_write(phydev, MDIO_DEVAD_NONE, MII_CTRL1000,
-					adv);
-
-			if (err < 0)
-				return err;
-			changed = 1;
-		}
-	}
-
-	return changed;
+    u32 advertise;
```

Comment:
I think this code was pulled from uboot and shouldn't be restyled

Reply:
It's hard to argue with the automatic style checker, it will always complain about this file for each commit that changes one line. That is why I made one commit that fixes the style now. I have some doubts this is every really synced with U-Boot again, as manche thing see completely different by now in the head version.

Reply:
You can add the file to `.stylefilter`, which if it doesn't exist yet should be at the repo root. This will cause the checker to skip the checks for this file. If it has diverged a lot already that might not be worth it any more, but if not it's an option.

Reply:
It's up to you to demand this. But I'd prefer no to rework this patch again, since the code has been reworked a lot already and diverges from U-Boot. There are also follow-up patches for the Nitrogen board that I have to rework then.

Reply:
Does it really matter that much if there are slight style changes?

Reply:
Given the code is from U-boot it's best to keep it as close to original as possible to pull in updates if needed. Ideally we'd check U-boot mainstream to see if these patches already exist there.